src/components/HomepageFeatures/styles.module.css

```diff
@@ -10,4 +10,13 @@
   /*adjust the size for the new svgs*/
   height: 100px;
   width: 100px;
+  stroke-width: 0.08px;
+  fill: #f5f6f7;
+  stroke: #f5f6f7;
 }
+
+/* Dark mode */
+[data-theme='light'] .featureSvg {
+  fill: #1C1E21;
+  stroke: #1C1E21; /* Change color in dark mode */
+}
```

Comment:
Comments say dark mode but these values are for "light" mode, right? I think just the comments need updating.

Reply:
Yep, seems like an oversight. Fixed now.

Reply:
Hi @DianaLease. I messed something up while trying to add the DCO signoff to my commit. I ended up created a [new PR](https://github.com/accordproject/concerto-docs/pull/41), complete with the suggested changes and the signoff. I hope it resolves the issue. Thanks for your patience.